src/test/scala/org/graphframes/lib/PageRankSuite.scala

```diff
@@ -46,4 +53,62 @@ class PageRankSuite extends SparkFunSuite with GraphFrameTestSparkContext {
     assert(gRank === 0.0,
       s"User g (Gabby) doesn't connect with a. So its pagerank should be 0 but we got $gRank.")
   }
+
+  test("Star example parallel personalized PageRank") {
+    val g = Graphs.star(n)
+    val resetProb = 0.15
+    val numIter = 10
+    val vertexIds: Array[Any] = Array(1L, 2L, 3L)
+
+    lazy val prc = g.parallelPersonalizedPageRank
+      .numIter(numIter)
+      .sources(vertexIds)
+      .resetProbability(resetProb)
+
+    if (isLaterVersion("2.1")) {
+      val pr = prc.run()
+      TestUtils.testSchemaInvariants(g, pr)
+      TestUtils.checkColumnType(pr.vertices.schema, "pagerank", SQLDataTypes.VectorType)
+      TestUtils.checkColumnType(pr.edges.schema, "weight", DataTypes.DoubleType)
+    } else {
+      intercept[NotImplementedError] { prc.run() }
+    }
+  }
+
+  test("friends graph with parallel personalized PageRank") {
+    val g = Graphs.friends
+    val resetProb = 0.15
+    val numIter = 10
+    val vertexIds: Array[Any] = Array("a")
+    lazy val prc = g.parallelPersonalizedPageRank
+      .numIter(numIter)
+      .sources(vertexIds)
+      .resetProbability(resetProb)
+
+    if (isLaterVersion("2.1")) {
+      val pr = prc.run()
+      val prInvalid = pr.vertices
+        .select("pagerank")
+        .collect()
+        .filter { row: Row =>
+          vertexIds.size != row.get(0).asInstanceOf[SparseVector].size
```

Comment:
Write ```row.getAs[Vector](0)``` instead since that will print a nicer error message when the type cast does not work.